### PR TITLE
Remove any references and tooling around Redis

### PR DESCRIPTION
**Pull request by deividaspetraitis**
This pull request removes all references and tooling around no longer used Redis, for more details please see #240.

In the documentation for the most scenarios ( except one in the `Changelog.md` ) `Redis` was simply replaced with `cache` in order to maintain original ideas and to make it consistent with current code base, but please let me know if I have missed anything or I should rephrase sentences in some places as I am knew on this project and might lack having some context.

Code related with the Redis were simply removed, and tests rerun to make sure nothing was broken.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 This is a sidecar query server that is used for performing query tasks outside of the main chain.
 The high-level architecture is that the chain reads data at the end of the block, parses it
-and then writes into a Redis instance.
+and then writes into a store.
```
Comment (p0mvn) [inline README.md]:
```suggestion
and then stores it in RAM.
```

File: start [hunk 2]
```diff
@@ -20,4 +18,4 @@ if [ "${DEBUG}" == "true" ]; then
   /bin/sqsd || echo "failed to start" && tail -f status
 else
   /bin/sqsd
-fi
\ No newline at end of file
+fi
```
Comment (p0mvn) [inline start]: This file can be removed. Do you mind doing this drive-by clean-up? 🙏

File: start.sh [hunk 3]
```diff
@@ -20,4 +18,4 @@ if [ "${DEBUG}" == "true" ]; then
   /bin/sqsd || echo "failed to start" && tail -f status
 else
   /bin/sqsd
-fi
\ No newline at end of file
+fi
```
Comment (p0mvn) [inline start.sh]: Ditto for removal

**Review by p0mvn — APPROVED** [review]
Thank you!

**Comment by deividaspetraitis** [comment]
Thank you for your input! I have updated code as per you suggestions and marked converstations as resolved.

**Review by p0mvn — APPROVED** [review]
Thank you!

**Comment by deividaspetraitis** [comment]
By the way, I am unable to merge, only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests. :)
